release-notes/8.0/preview/api-diff/preview5/Microsoft.WindowsDesktop.App/8.0-preview5_System.Windows.Forms.md

```diff
@@ -0,0 +1,301 @@
+# System.Windows.Forms
+
+``` diff
+ namespace System.Windows.Forms {
+     public class AutoCompleteStringCollection : ICollection, IEnumerable, IList {
+-        bool System.Collections.IList.IsFixedSize { get; }
++        bool IList.IsFixedSize { get; }
```

Comment:
@carlossanlop, this is coming again. i remember you made changes to ignore only namespace optimization changes in the diff.

Reply:
Could be another manifestation of the same bug. I'll take a look.